charts/ndb-operator/values.yaml

```diff
@@ -3,8 +3,6 @@ replicaCount: 1
 image:
   repository: ghcr.io/nutanix-cloud-native/ndb-operator/controller
```

Comment:
Even the repo URL should go? No point letting the users specify the docker registry if we are removing the appVersion already.

Reply:
same private registry and airgap need ability to change repo